contracts/src/PDPService.sol

```diff
@@ -225,9 +230,15 @@ contract PDPService {
         return rootId;
     }
 
+    function enqueueRemovals(uint256 setId, uint256[] calldata rootIds) public {
+        for (uint256 i = 0; i < rootIds.length; i++){
+            enqueuedRemovals[setId].push(rootIds[i]);
```

Comment:
TODO: add application notify call.  This is the only place for access control on removal since once its already slotted for removal proving won't be possible since prove and remove happen atomically.

Reply:
TODO: probably want limits on how many removals we can enqueue so that proof of possession doesn't run out of gas